apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -57,16 +57,20 @@ const answerDataPropType = PropTypes.shape({
   isCorrectAnswer: PropTypes.bool,
 });
 
-const questionDataPropType = PropTypes.shape({
+const multipleChoiceDataPropType = PropTypes.shape({
   id: PropTypes.number.isRequired,
   question: PropTypes.string.isRequired,
   answers: PropTypes.arrayOf(answerDataPropType),
   notAnswered: PropTypes.number.isRequired,
 });
 
+/* A single table that shows students' responses to each multiple choice question.
```

Comment:
your multi line comments are missing some `*`s. 

Check out `starred-block` in this link: https://eslint.org/docs/rules/multiline-comment-style